framework/include/cppmicroservices/detail/BundleAbstractTracked.tpp

```diff
@@ -51,7 +51,7 @@ void BundleAbstractTracked<S,TTT,R>::SetInitial(const std::vector<S>& initiallis
     for(typename std::list<S>::const_iterator item = initial.begin();
       item != initial.end(); ++item)
     {
-      DIAG_LOG(*bc->GetLogSink()) << "BundleAbstractTracked::setInitial: " << (*item);
+      DIAG_LOG(*bc->GetLogSink()) << "BundleAbstractTracked::setInitial: " << (*item) << "\n";
```

Comment:
> Can we incorporate the "\n" into the DIAG_LOG macro or the logsink.

Yes, I'll do that.

>  can we use std::endl instead of "\n"

I'd prefer to use ```\n``` and not flush the stream after each log message.